Irina/src/main/java/oop/AdminUser.java

```diff
@@ -0,0 +1,22 @@
+package oop;
+
+public class AdminUser extends AbstractUser {
+    private final String adminUsername;
```

Comment:
Why do you need adminUsername when you have field name in parent class?

Reply:
Done